silk/views/requests.py

```diff
@@ -130,7 +130,7 @@ def _get_objects(self, show=None, order_by=None, order_dir=None, path=None, filt
         return query_set[:show]
 
     def _create_context(self, request):
-        raw_filters = request.session.get(self.session_key_request_filters, {}).copy()
+        raw_filters = request.session.get(self.session_key_request_filters, {}).copy() if hasattr(request, 'session') else {}
```

Comment:
Thanks for your work, it's a great beginning, but if I'm not mistaking, the filters are not working when selectining them. They should work if you fall back to `request.GET or request.POST` here